content/about/community/join/index.md

```diff
@@ -10,23 +10,30 @@ to join us and get involved!
 There are quite a few ways to get plugged in:
 
 {{< community_item logo="./discourse.svg" alt="Discourse" >}}
-Head for the [Istio discussion board](https://discuss.istio.io) to participate in discussions
-and get help troubleshooting problems.
+Head for the [Istio discussion board](https://discuss.istio.io) to ask questions, 
+participate in discussions, and get help troubleshooting problems. You can use your github ID to login!
```

Comment:
I don't know if you care, but users can also use a Google account, Twitter, or Yahoo.
'github' -> GitHub.

Reply:
Yes will do.